frontend/src/static/js/v_authorship.js

```diff
@@ -372,6 +385,30 @@ window.vAuthorship = {
       },
     },
 
+    isBinaryChecked: {
+      get() {
+        return this.isBinaryFilesChecked;
+      },
+      set(value) {
+        if (value) {
+          this.isBinaryFilesChecked = true;
+        } else {
+          this.isBinaryFilesChecked = false;
+        }
+
+        this.updateSelectedFiles();
+        this.indicateCheckBoxes();
+      },
+    },
+
+    selectedFiles() {
```

Comment:
make this either data or computed property

Reply:
Actually I think this section is redundant and I will remove it accordingly.